modules/aioseop_sitemap.php

```diff
@@ -3849,10 +4068,10 @@ public function get_tax_args( $taxonomy, $page = 0 ) {
 		 *
 		 * @return mixed
 		 */
-		public function set_post_args( $args ) {
-			if ( $this->option_isset( 'excl_categories' ) ) {
+		public function set_post_args( $args, $only_tax = false ) {
```

Comment:
PHPDoc will need to be updated, and a `since` added noting the change.

Reply:
done